test/todo-app/src/components.d.ts

```diff
@@ -15,6 +15,14 @@ export namespace Components {
         "text": string;
     }
 }
+export interface TodoInputCustomEvent<T> extends CustomEvent<T> {
```

Comment:
Did this change get triggered by other things in the PR? Or is this a case of us (me) merging `main` into this branch and not running a build?

Reply:
oh right, I meant to point this out / ask

I had some test failures I had to work through with the `test.analysis` npm script, and running it locally led to these changes. Should we commit them?

Reply:
Ah, I see what happened here. These should have been committed a while ago 😨 

My vote is we split them out into a separate PR. Can you do me a favor and create a branch off of `v3.0.0-dev` and run `npm run build` in the `test` dir? That should be enough to replicate these changes. While we're there, it may make sense to propagate our check for a dirty git context over to the main CI workflow (the irony being, IIRC, that's where we originally had it in the first implementation PR before I asked for it only to be in Browserstack 🤦 )

Reply:
haha ok sure, will do, and I'll delete the file here 👍

Reply:
https://github.com/ionic-team/stencil/pull/3582